### DISCUSSION TITLE
[wagmi/viem] Include the RPC used in the error message

### DISCUSSION THREAD

**Discussion by kasparkallas** [wevm/wagmi#4341] [Ideas]
Sometimes contract simulations and gas estimations fail with HTTP status codes (i.e. 429 Too Many Requests). With the use of the [fallback](https://wagmi.sh/core/api/transports/fallback#fallback) Transport, it's not always obvious which RPC is the flaky problematic one.

The error message already contains the version of wagmi/viem. I think it could be helpful to include the RPC used for the contract call as well.

**Comment by tmm** [comment]
Good idea. Best to suggest this on the Viem repo.
